### PR TITLE
nikolai.berestevich/refactor/ alt tag delete on about us pages

### PR DESCRIPTION
**Pull request by nikolaiberestevich-deriv**
Changes:

-   alt tags  were changed to empty on "about us" pages

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ x ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/careers/_employee-testimonial-carousel.js [hunk 1]
```diff
@@ -74,20 +74,14 @@ const QuoteWrapper = styled(Flex)`
...
-                                loading="lazy"
-                                width="100%"
-                                height="100%"
-                                src={img_path}
-                                alt={img_alt}
-                            />
+                            <img loading="lazy" width="100%" height="100%" src={img_path} alt="" />
                         </ImageWrapper>
                     </Flex>
                     <QuoteWrapper direction="column">
```
Comment (njazuli-deriv) [inline src/pages/careers/_employee-testimonial-carousel.js:87]: there's one more here 
'quotemark' line 88

Comment (nikolaiberestevich-deriv) [reply src/pages/careers/_employee-testimonial-carousel.js:87]: I didn't delete it because this image doesn't have any description

Comment (njazuli-deriv) [reply src/pages/careers/_employee-testimonial-carousel.js:87]: Noted on this..

**Review by njazuli-deriv — APPROVED** [review]

File: src/pages/careers/_employee-testimonial-carousel.js [hunk 2]
```diff
@@ -111,28 +104,24 @@ EmployeeSlide.propTypes = {
 
 const ahmad = {
     name: 'Ahmad Aizuddin, Disaster Recovery Analyst',
-    img_alt: 'Ahmad - Recovery Analyst',
```
Comment (sean-binary) [inline src/pages/careers/_employee-testimonial-carousel.js:114]: @nikolaiberestevich-deriv We will need these as this one is for the image itself and not an icon.

File: src/pages/careers/locations/index.js [hunk 3]
```diff
@@ -91,13 +91,13 @@ const CountryCardWrapper = styled(StyledCard)`
 const CountryCard = ({ country_name, city_name, link, img_data, Icon }) => {
     return (
         <CountryCardWrapper to={link}>
-            <QueryImage data={img_data} alt={city_name + localize('Office')} width="100%" />
```
Comment (sean-binary) [inline src/pages/careers/locations/index.js:94]: Same for this one as well, we will need this as these are pure images and not icons. We need a fallback text for this.

File: src/pages/contact_us/_contact-ways.js [hunk 4]
```diff
@@ -117,11 +117,7 @@ const ContactWays = () => {
                         return (
                             <ContactWrapper key={idx}>
                                 <ImgWrapper>
-                                    <QueryImage
```
Comment (sean-binary) [inline src/pages/contact_us/_contact-ways.js:120]: Here too.

File: src/pages/contact_us/_offices.js [hunk 5]
```diff
@@ -199,11 +199,7 @@ export const Offices = () => {
                                         target="_blank"
                                         rel="noopener noreferrer"
                                     >
-                                        <QueryImage
-                                            alt="Map Malta"
-                                            data={data['map_malta']}
-                                            height="100%"
-                                        />
```
Comment (sean-binary) [inline src/pages/contact_us/_offices.js:206]: Same here and the other maps for other countries as well.

File: src/pages/leadership/_leaders.js [hunk 6]
```diff
@@ -309,7 +309,7 @@ const Leaders = () => {
                         {leaders_data.map((leader, index) => (
                             <LeaderWrapper key={index} direction="column" ai="center">
                                 <Box margin={{ bottom: '0.8rem' }} width="100%" position="relative">
-                                    <QueryImage data={data[leader.image]} alt={leader.name} />
```
Comment (sean-binary) [inline src/pages/leadership/_leaders.js:312]: Same here and for the other image below.

**Review by sean-binary — CHANGES_REQUESTED** [review]

**Review by sean-binary — APPROVED** [review]

**Pull request merged**